components/brave_wallet_ui/utils/block-explorer-utils.ts

```diff
@@ -86,7 +84,7 @@ export const openBlockExplorerURL = ({
     const url = buildExplorerUrl(network, type, value, id)
 
     if (!chrome.tabs) {
-      window.open(url, '_blank')
+      window.open(url, '_blank', 'noopener')
```

Comment:
noreferrer is actually slightly better since it implies noopener but either is fine. thanks.

Reply:
Thank you! Switched to `noreferrer`! https://github.com/brave/brave-core/compare/05c6fda7f0ccc5b763e99d5e22f432b7050e1efe..efe1394b80d2a4ae52aaa39f5f1016dec183a5ad